src/dom.js

```diff
@@ -974,7 +974,7 @@ export function getVerticalScrollbarWidth(win) {
 export function dispatchCustomEvent(node, name, opt_data, opt_options) {
   const data = opt_data || {};
   // Constructors of events need to come from the correct window. Sigh.
-  const event = node.ownerDocument.createEvent('Event');
+  const event = /** @type {?}*/ (node.ownerDocument.createEvent('Event'));
```

Comment:
Why are we typecasting this to an unknown type?

Reply:
I narrowed the `?` casting to just the problematic assignment.  `Event` doesn't have a `.data`